tests/system/small/bigquery/test_struct.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+
+import bigframes.bigquery as bbq
+import bigframes.series as series
+
+
+def test_struct_from_dataframe():
```

Comment:
I am curious the following cases work for `bbq.struct(df)`
1. When the `df` has a `struct` type column.
2. When the `df` has a `int` column, which has a `None` element. 
3. When the `df` has a `array` type column. 
If they're working, could you please add more tests for them?

Reply:
done!